src/current/_includes/releases/v25.2/feature-detail-key.html

```diff
@@ -7,11 +7,11 @@
  <tbody>
   <tr>
    <td>&starf;</td>
-   <td>Features marked "All&starf;" were recently made available in the CockroachDB Cloud platform. They are available for all supported versions of CockroachDB, under the deployment methods specified in their row under Availability.</td>
+   <td>Features marked "All*" were recently made available in the CockroachDB Cloud platform. They are available for all supported versions of CockroachDB, under the deployment methods specified in their row under Availability.</td>
```

Comment:
Use the 'cloud' liquid var here?

Reply:
It's included HTML so I'm wary; I'll make a note to test that change post-release